### PR TITLE
test script loading

### PR DESCRIPTION
**Pull request by atk**
It's known to be impossible to test actual script loading with vitest and jsdom, so, naturally, I tried to work around that issue.

### CODE REVIEW

File: packages/script-loader/package.json [hunk 1]
```diff
@@ -42,6 +42,10 @@
     "test": "vitest -c ../../configs/vitest.config.ts",
     "test:ssr": "pnpm run test --mode ssr"
   },
+  "devDependencies": {
+    "@types/jsdom": "^21.1.1",
```
Comment (thetarnav) [inline packages/script-loader/package.json:46]: Those should probably be added to the root package.json do they are available in each package
`jsdom` is already added this way

Comment (atk) [reply packages/script-loader/package.json:46]: I know, but I get a TS error, since it cannot find the dependencies in the local package.json.

Comment (thetarnav) [reply packages/script-loader/package.json:46]: I mean it has to be just the types, they are not added to root package.json. Might as well at it there. It should be available just as `vitest` it

Comment (atk) [reply packages/script-loader/package.json:46]: Let me try tomorrow.

**Pull request merged**